<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Telemetry</class>
 <widget class="QMainWindow" name="Telemetry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Telemetry</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{border:0px solid white; background-color: rgba(0,0,0,0);}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="m_lblTitle">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="toolTipDuration">
       <number>0</number>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QToolButton" name="m_btnStart">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QToolButton" name="m_btnFriends">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton::checked{
background-color:rgba(0,0,0,0);
}</string>
      </property>
      <property name="text">
       <string>Display All</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QTableWidget" name="m_tblTimes">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget{background-color: rgba(0, 0, 0, 0);
border:0px solid white;
color: white;}

QTableCornerButton::section {border: 0px solid white; background: rbga(0,0,0,0);}

QHeaderView, QHeaderView::section {color: white; background-color: rgba(0,0,0,0); font-weight: bold; font-size: 14px; height: 30; width: 30; border:0px solid white;}</string>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="autoScrollMargin">
       <number>16</number>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::NoPen</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>6</number>
      </property>
      <property name="columnCount">
       <number>8</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
